servers/rendering/renderer_rd/storage_rd/texture_storage.cpp

```diff
@@ -73,6 +73,11 @@ TextureStorage *TextureStorage::get_singleton() {
 	return singleton;
 }
 
+void TextureStorage::_bind_methods() {
+	ClassDB::bind_method(D_METHOD("decal_atlas_get_texture"), &TextureStorage::decal_atlas_get_texture);
+	ClassDB::bind_method(D_METHOD("decal_atlas_get_texture_srgb"), &TextureStorage::decal_atlas_get_texture_srgb);
+}
```

Comment:
@BastiaanOlij not 100% percent clear what is the difference between
decal_atlas_get_texture and decal_atlas_get_texture_srgb methods, cuz in the class it's
```cpp
      decal_atlas.texture = RD::get_singleton()->texture_create(tformat, RD::TextureView(), vpv);
      decal_atlas.texture_srgb = decal_atlas.texture;
```